### ISSUE TITLE
NTR: brain regions

### ISSUE THREAD

**Issue by paulacalipho** [obophenotype/uberon#2832]
Term: left fronto parietal lobe
Synonyms: 
exact: left fronto-parietal lobe PMID:3703146, PMID:27003199, PMID:23808679, PMID:30797641
Definition: The left frontal and parietal lobes of the brain hemisphere.
SubClass Of: ‘organ part’ UBERON:0000064 and ('part of' some ‘left cerebral hemisphere’ UBERON:0002812)
https://orcid.org/0000-0002-0819-0473

Term: right fronto parietal lobe
Synonyms: 
exact: right frontal parietal lobe PMID: 29678709
exact: right frontal-parietal lobe PMID: 22568892
exact: right fronto-parietal lobe PMID: 34234736, PMID:34736485, PMID:30169667
Definition: The right frontal and parietal lobes of the brain hemisphere.
SubClass Of: ‘organ part’ UBERON:0000064 and ('part of' some ‘right cerebral hemisphere’ UBERON:0002813)
https://orcid.org/0000-0002-0819-0473

Term: left occipital parietal lobe
Synonyms: 
exact: left parietal and occipital lobes PMID:30383697
exact: left parietal-occipital lobe PMID:22799762, PMID:23506194
exact: left parieto-occipital lobe PMID:27028405, PMID:36462980, PMID:36542021, PMID:36198519, PMID:36451532, PMID:36090026, PMID:25389451
broad: left occipital-parietal region PMID: 26300758, PMID:17215478, PMID:35257944
broad: left occipital/parietal region PMID:36561241, PMID:22078521
broad: left occipital parietal region PMID:26027887
broad: left occipito-parietal region PMID:27303466
Definition: The left occipital and parietal lobes of the brain hemisphere.
SubClass Of: ‘organ part’ UBERON:0000064 and ('part of' some ‘left cerebral hemisphere’ UBERON:0002812)
Mapping to other ontologies: Left parietal and occipital lobes, CS C1267724
https://orcid.org/0000-0002-0819-0473

Term: right occipital parietal lobe
Synonyms: 
broad: right occipital-parietal region PMID:35257944
broad: right occipital parietal region PMID:31053785, PMID:22711620
Definition: The right occipital and parietal lobes of the brain hemisphere.
SubClass Of: ‘organ part’ UBERON:0000064 and ('part of' some ‘right cerebral hemisphere’ UBERON:0002813)
Mapping to other ontologies: Right parietal and occipital lobes CS C1267723
https://orcid.org/0000-0002-0819-0473

Term: left temporo occipital lobe
Synonyms: 
exact left temporo-occipital lobe PMID: 33452388, PMID:24366606
exact occipitotemporal lobe PMID: 36093544, PMID: 29077534, PMID: 14983026
broad L; occipitotemporal PMID:23951083
Definition: The left temporal and occipital lobes of the brain hemisphere.
SubClass Of: ‘organ part’ UBERON:0000064 and ('part of' some ‘left cerebral hemisphere’ UBERON:0002812)
https://orcid.org/0000-0002-0819-0473

Term: right temporo occipital lobe
Synonyms: 
exact: right temporo-occipital lobe PMID:34595871, PMID:24299901, PMID:31036340, PMID:15883270, PMID:36626518, PMID:27489862, PMID:27250577, PMID:21219302, PMID:31068568
Definition: The right temporal and occipital lobes of the brain hemisphere.
SubClass Of: ‘organ part’ UBERON:0000064 and ('part of' some ‘right cerebral hemisphere’ UBERON:0002813)
https://orcid.org/0000-0002-0819-0473

Term: left temporo parietal lobe
Synonyms: 
exact: left temporal-parietal lobe PMID: 34858900, PMID: 31903645
exact: left temporo-parietal lobe PMID: 28711476, PMID: 27324529, PMID: 21453038, PMID: 9652669
broad: left temporo-parietal PMID:8154317
Definition: The left temporal and parietal lobes of the brain hemisphere.
SubClass Of: ‘organ part’ UBERON:0000064 and ('part of' some ‘left cerebral hemisphere’ UBERON:0002812)
https://orcid.org/0000-0002-0819-0473

Term: right temporo parietal lobe
Synonyms: 
exact: right temporoparietal lobe PMID:35795795
exact: right temporo-parietal lobe PMID:21112225, PMID:30797641, PMID:33787635, PMID:35331158
broad: right temporo-parietal region PMID:21112225
Definition: The right temporal and parietal lobes of the brain hemisphere.
SubClass Of: ‘organ part’ UBERON:0000064 and ('part of' some ‘right cerebral hemisphere’ UBERON:0002813)
https://orcid.org/0000-0002-0819-0473

Term: right parasagittal parietal lobe
Synonyms: 
broad: right parasagittal parietal region PMID:23835274
Definition: The right parasagittal parietal region of the brain hemisphere.
SubClass Of: ‘organ part’ UBERON:0000064 and ('part of' some ‘right cerebral hemisphere’ UBERON:0002813)
https://orcid.org/0000-0002-0819-0473

Term: right periatrial region of the brain
Synonyms: 
exact: peri-atrial brain region PMID:25304435, PMID:28834867
exact: right periatrial region PMID:11498414, PMID:36340223, PMID:36324946, PMID:23859284, PMID:30937235, PMID:31133966, PMID:12748099, PMID:27028405, PMID:18417739
exact: occipitotemporal periventricular right hemisphere PMID:26744442
Definition: -
SubClass Of: ‘organ part’ UBERON:0000064 and ('part of' some ‘right cerebral hemisphere’ UBERON:0002813)
https://orcid.org/0000-0002-0819-0473

Term: medial temporal lobe
Definition: The medial temporal lobe includes the hippocampus, amygdala and parahippocampal regions, and is crucial for episodic and spatial memory (PMID:28428747)
SubClass Of: ‘organ part’ UBERON:0000064 and ('part of' some ‘temporal lobe’ UBERON:0001871)
Mapping to other ontologies: BTO:0004477
Term: left temporo-medial region
Synonyms: 
broad: left temporal-medial region PMID: 28144495
broad: left temporal medial region PMID: 28144495
Definition: The left medial temporal lobe of the brain hemisphere.
SubClass Of: ‘organ part’ UBERON:0000064 and ('part of' some 'medial temporal lobe' UBERON:0001871)
https://orcid.org/0000-0002-0819-0473

Term: right frontal parietal occipital cortex
Synonyms: 
exact: right frontal parieto-occipital cortex PMID:35372056, PMID:34045642, PMID:33375627, PMID:27003178
Definition: The right frontal, parietal, and occipital cortex of the brain hemisphere.
SubClass Of: ‘cortex of cerebral lobe' UBERON:0016529 and ('part of' some ‘right cerebral hemisphere’ UBERON:0002813)
https://orcid.org/0000-0002-0819-0473

Term: left prefrontal cortex
Synonyms: 
broad: left prefrontal lobe PMID:35964192, PMID:35556171, PMID:34483857
Definition: The left prefrontal cortex of the brain.
SubClass Of: ‘prefrontal cortex' UBERON:0000451 and ('part of' some ‘left cerebral hemisphere’ UBERON:0002812)
Mapping to other ontologies: Left prefrontal cortex FMA:242627
https://orcid.org/0000-0002-0819-0473

**Comment by bvarner-ebi** [comment]
@patrick-lloyd-ray, would you mind scanning these potential new terms and providing feedback on whether they would be appropriate classes in Uberon?

Some of the terms seem to represent brain regions instead of actual lobes ("right parasagittal parietal lobe") even though they may be referred to as lobes in literature. If these are appropriate to add to Uberon, do you have thoughts on the proposed modelling?

**Comment by patrick-lloyd-ray** [comment]
@bvarner-ebi You're right in that these classes labeled lobes are regions, not lobes. But, I think it would be helpful for UBERON to capture usage, so listing these as synonyms is fine if UBERON wants to add them. 

As far as _whether_ UBERON should add these terms, that's a decision for @cmungall and/or @dosumis. These regions cut across well defined structures and that raises some questions regarding design that should be coordinated going forward. 

IMO, I think that brain anatomy is complex in that there are many equally meaningful/important/useful parcellations (even in a single species). On one hand, it would be great to have all of these represented UBERON. On the other hand, it would be a ton of work.

**Comment by paulacalipho** [comment]
Dear @patrick-lloyd-ray  and @bvarner-ebi thanks for your feedback. I updated the NTR with the following:

- modified the term name from lobe to region
- changed some synonyms from exact synonym to related synonym
- replaced the SubClass from ‘lobe of cerebral hemisphere’ UBERON:0016526 to 'regional part of brain' UBERON:0002616
- adapted the definition

I let you decide the relevance of these terms in UBERON. Thanks, Paula

**Comment by bvarner-ebi** [comment]
Thanks for the feedback, @patrick-lloyd-ray.
Thanks for the modifications, @paulacalipho.

@cmungall, @dosumis:  appreciate your feedback on whether all these terms should be added as new classes.

**Comment by paulacalipho** [comment]
Hi @cmungall do you think these term are in the scope of UBERON?

**Comment by cmungall** [comment]
> IMO, I think that brain anatomy is complex in that there are many equally meaningful/important/useful parcellations (even in a single species). 

Definitely - but Uberon already has many alternate parcelations

> On one hand, it would be great to have all of these represented UBERON. On the other hand, it would be a ton of work

My main concern is the L/R ones, while it seems easy to add these keeping relationships consistent is hard, especially as new L/R requests come in. However, we really have to solve this by getting the patterns down..
